db/migrate/20240129215920_create_invoices.rb

```diff
@@ -0,0 +1,10 @@
+class CreateInvoices < ActiveRecord::Migration[6.1]
+  def change
+    create_table :invoices do |t|
+      t.string :invoice_number
+      t.integer :user_id
```

Comment:
typically this would be "t.belongs_to :user, foreign_key: true" to create the association, and ensure the user_id is here.

Reply:
Ah ok great! I couldnt figure out if the foreign key should be added to the migration file cos I only found examples of it being added in the model file so that's extremely helpful thanks :)